app/models/teacher_set.rb

```diff
@@ -1084,8 +1081,6 @@ def update_notes(teacher_set_notes_string)
   # Updated MLN DB with Total copies and available copies.
   def update_available_and_total_count(bibid)
     response = get_items_info_from_bibs_service(bibid)
-    LogWrapper.log('INFO','message' => "TeacherSet available_count: #{response[:available_count]}, total_count: #{response[:total_count]},
-    availability: #{response[:availability_string]}", b_number: "#{bibid}")
```

Comment:
available_count, total_count already logging in anther method. So removing this log